deployment/human-connection/deployment-neo4j.yaml

```diff
@@ -1,47 +1,61 @@
----
-  apiVersion: extensions/v1beta1
-  kind: Deployment
-  metadata:
-    name: nitro-neo4j
-    namespace: human-connection
-  spec:
-    replicas: 1
-    strategy:
-      rollingUpdate:
-        maxSurge: 0
-        maxUnavailable: "100%"
-    selector:
-      matchLabels:
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    human-connection.org/selector: deployment-human-connection-neo4j
+  name: neo4j
+  namespace: human-connection
+spec:
+  progressDeadlineSeconds: 2147483647
```

Comment:
Where does that come from?

Reply:
it came from the command `kubectl convert -f deployment-neo4j.yaml --output-version apps/v1`
when I tried to apply the config, it failed since the api has been updated from `extensions/v1beta1`